app/MMB/src/menuGroups/__tests__/ServicesMenuGroup.test.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+
+import { ServicesMenuGroup } from '../ServicesMenuGroup';
+import MenuItem from '../../components/menu/MenuItem';
+
+describe('ServicesMenuGroup', () => {
+  it('renders with future trip', () => {
+    const wrapper = renderer.create(
+      <ServicesMenuGroup
+        openSubmenu={jest.fn()}
+        activeId="123"
+        isPastBooking={false}
+      />,
+    );
+
+    expect(wrapper.root.findAllByType(MenuItem).length).toBe(2);
+  });
+
+  it('renders with past trip', () => {
+    const wrapper = renderer.create(
+      <ServicesMenuGroup
+        openSubmenu={jest.fn()}
+        activeId="123"
+        isPastBooking={true}
+      />,
+    );
+    expect(wrapper.root.findAllByType(MenuItem).length).toBe(1);
```

Comment:
No snapshots - very nice! 👏 Today is your day... 😊

It may be a little bit wonky in the future but I like it anyway.

Reply:
I do agree that snapshots are useless, trying to find the better ways to test with react-test-renderer 🙂